src/course-unit/data/api.js

```diff
@@ -9,6 +9,7 @@ import {
   appendBrowserTimezoneToUrl,
   normalizeCourseSectionVerticalData,
 } from './utils';
+import { PUBLISH_TYPES } from '../constants';
```

Comment:
[nit] Let's move it above of './utils' import.

Reply:
Moved, thanks